packages/patternfly-4/react-core/src/components/Popover/Popover.tsx

```diff
@@ -95,7 +95,7 @@ export interface PopoverProps {
   /** z-index of the popover */
   zIndex?: number;
   /** additional Props to pass through to tippy.js */
-  tippyProps?: TippyProps;
+  tippyProps?: object;
```

Comment:
Are Tippy's types no longer working? Avoiding `object` would be nice for consumers.

Reply:
there are like 43 required TippyProps. So in the tests, for example, we tried only passing two tippy props, we get a type error.

Reply:
Does `Partial<TippyProps>` work?